internal/validation/extension.go

```diff
@@ -122,14 +123,28 @@ func (s *Server) Process(srv pb.ExternalProcessor_ProcessServer) error {
 				}
 				err = s.validate(req, service, operation)
 				if err != nil {
+					errMsg := ErrorBody{}
+					errMsg.SetErrorBody(err)
+
 					resp = &pb.ProcessingResponse{
 						Response: &pb.ProcessingResponse_ImmediateResponse{
 							ImmediateResponse: &pb.ImmediateResponse{
 								Status: &v32.HttpStatus{Code: v32.StatusCode_BadRequest},
-								Body:   err.Error(),
+								Body:   errMsg.Error,
+								Headers: &pb.HeaderMutation{
+									SetHeaders: []*v31.HeaderValueOption{
+										{
+											Header: &v31.HeaderValue{
+												Key:   "Content-Type",
```

Comment:
Make this a constant please, it is repeated elsewhere.

Reply:
done